### DISCUSSION TITLE
Customizing v1.2.5

### DISCUSSION THREAD

**Discussion by Stanseas** [Yue-Yang/ChatGPT-Siri#120] [Q&A] [answered]
Thank you for the Shortcut.

At first it didn't work for me so as I inspected the details there were a number of instances where the word "title" was misspelled "tittle". Fixing those got it working. 

I tried to customize the wording of all the alerts but even though I changed all I could find it still showed the old message used to initiate a follow-up question in a single session instead of mine. Where is it pulling that text from? The words are not visible in the lines of code I can see. It must be buried somewhere but I can't find it.

I'm also interested in changing the input from text to dictated-text and have the results spoken by Siri. I haven't nailed down exactly where to change that function. I also have a way to add a sound to the prompt but I don't know where that initial question is generated to add it. It's likely these changes are far more complicated than I realize, but I'm learning.

If there was a way to view the code in a text editor I'd be able to figure it out on my own, but at this point any pointers would be appreciated. Thank you.

**Comment by dshk0718** [comment, marked as answer]
So after hours of tweaking, I've managed to get the previous response to stay before Siri asks for a new prompt for ChatGPT:
[ChatGPT 1.2.5-tweaked](https://www.icloud.com/shortcuts/a9c7cc0151bc44368241a67a02f72cf7)
This is a shortcut that I've tweaked from Yue-Yang's original shortcut version 1.2.5 and the only difference is the fact that the ChatGPT output is used to ask for the next prompt (instead of the continue-text), which is actually similar to the behavior that the ChatGPT web app on OpenAI does (if you have used it on OpenAI site, you would know what I mean). With these tweaks, the GPT responses stay on the screen, which is very helpful in my opinion. Also, I've fixed some minor spelling errors within the shortcut.
I hope this helps!

Here is the example session:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/22249997/233913785-60fa1bfa-9347-4f69-9c18-15e07088b3d1.png">

The first question was: "What is a breadth-first search?".
The second question was: "What is a depth-first search?".
The last question was: "What is the major difference between the two?".

As for the actual output you are looking for, it's named `chatgpt-output` toward the bottom of the shortcut code.
Also, if you want to use the type-to-text instead of speech-to-text, go to Settings -> Accessibility -> Siri and turn on the `Type to Siri` option. For results to always be spoken by Siri, go to Settings -> Siri & Spotlight -> Siri Responses and turn on the `Voice feedback` option.

As for adding the sound to the prompt, use the right-hand side toolbar to search the "Play Sound" module and drag and drop that module before the "Ask for `Text` with `input-title`" module.
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/22249997/233916092-1cf804db-2939-42c7-a9fa-2f6960b421af.png">
